### PR TITLE
docs: add model serving feature

### LINKED ISSUE #42: Add model-serving feature in WebUI Docs
Related: [webui#1900](https://github.com/lablup/backend.ai-webui/issues/1900)
After merging the related PR (model-serving), we need to provide proper documentation for model-serving accordingly.
In order to make user understand what is model-serving, how to use it, and how to test whether their service is successfully created, we need to describe several things below:

1. Brief definition of model-serving in Backend.AI
2. How to create model type folder
3. How to create/terminate model service (including configuration)
4. How to generate token to limit the end-user by authorization
5. How to edit configuration of model service

### PR DESCRIPTION
**Pull request by lizable**
This PR resolves #42.

### CODE REVIEW

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 1]
```diff
@@ -0,0 +1,961 @@
...
+#: ../../model_serving/model_serving.rst:11
+msgid ""
+"Backend.AI not only facilitates the construction of development "
+"environments and resource management during the model training phase, but"
+" also supports the model service feature from version 23.09 onwards. This"
+" feature allows end-users (such as AI-based mobile apps and web service "
+"backends) to make inference API calls when they want to deploy the "
+"completed model as an inference service."
+msgstr ""
+"Backend.AI(백엔드닷에이아이)에서는 모델 학습 단계의 개발 환경 구축 및 자원 관리를 쉽게 해주는 것 뿐 만 아니라,모델을 "
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:41]:
```suggestion
"Backend.AI(백엔드닷에이아이)에서는 모델 학습 단계의 개발 환경 구축 및 자원 관리를 쉽게 해주는 것뿐만 아니라, 모델을 "
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 2]
```diff
@@ -0,0 +1,961 @@
...
+"the inference session. It is stored in the model type folder "
+"independently from the container image that contains the inference "
+"service engine. This allows the engine to serve different models based on"
+" specific requirements and eliminates the need to build and deploy a new "
+"container image every time the model changes. By loading the model "
+"definition and model data from the network storage, the deployment "
+"process can be simplified and optimized during automatic scaling."
+msgstr ""
+"모델 정의 파일은 Backend.AI 시스템이 추론용 세션을 자동으로 시작, 초기화하고 필요에 따라 스케일링할 때 필요한 설정 "
+"정보를 담고 있는 파일입니다. 이 파일은 추론 서비스 엔진을 담고 있는 컨테이너 이미지와는 독립적으로 모델 타입 폴더에 저장합니다."
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:182]:
```suggestion
"정보를 담고 있는 파일입니다. 이 파일을 추론 서비스 엔진을 담고 있는 컨테이너 이미지와는 독립적으로 모델 타입 폴더에 저장합니다."
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 3]
```diff
@@ -0,0 +1,961 @@
...
+
+#: ../../model_serving/model_serving.rst:110
+msgid ""
+"Values marked as required in the notes must be included in the model "
+"definition file. Other items are optional and can be omitted.Items with a"
+" ``/`` slash indicate descriptions for sub-keys under the key preceding "
+"the slash."
+msgstr ""
+"비고에 필수항목이라고 적혀있는 값은 모델 정의 파일에 반드시 추가되어야 하는 항목입니다. 그 외의 항목은 입력하지 않아도 "
+"무방합니다.``/``(슬래시) 가 추가된 항목은 슬래시 앞 키의 하위 키에 대한 설명입니다."
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:203]: image

```suggestion
"무방합니다.``/`` (슬래시) 가 추가된 항목은 슬래시 앞 키의 하위 키에 대한 설명입니다."
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 4]
```diff
@@ -0,0 +1,961 @@
...
+msgid "``-``"
+msgstr "``-``"
+
+#: ../../model_serving/model_serving.rst:132
+msgid "``service/pre_start_actions``"
+msgstr "``service/pre_start_actions``"
+
+#: ../../model_serving/model_serving.rst
+msgid "preceding commands"
+msgstr "``start_command``항목 내"
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:299]:
```suggestion
msgstr "``start_command`` 항목 내"
```

**Review by agatha197 — CHANGES_REQUESTED** [review]
image
When I tested model serving a few days ago, if the `pre_start_actions` option is an empty string, it occurs error. Please fill it with `[]` (maybe?) or remove it.

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 5]
```diff
@@ -0,0 +1,961 @@
...
+
+#: ../../model_serving/model_serving.rst:269
+msgid ""
+"Desired Routing Count: The model service can be serviced by multiple "
+"servers. This setting determines how many routing sessions to create for "
+"the current service. The value specified here will be used as the basis "
+"for creating the sessions."
+msgstr ""
+"Desired Routing Count: 모델 서비스에서 서비스하고자 하는 서버가 여러개가 될 수 있습니다. 이 때 라우팅, 현재 "
+"기준으로 서비스하려는 세션을 몇 개까지 만들어야 할 지 설정하는데, 이 값을 기준으로 하게 됩니다."
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:752]:
```suggestion
"기준으로 서비스하려는 세션을 몇 개까지 만들어야 할지 설정하는데, 이 값을 기준으로 하게 됩니다."
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 6]
```diff
@@ -0,0 +1,961 @@
...
+"the Backend.AI scheduler. Therefore, it is recommended to terminate the "
+"model service if it is no longer needed. To terminate the model service, "
+"click on the trash icon in the Control tab. A modal will appear asking "
+"for confirmation to terminate the model service. Clicking ``OK`` will "
+"terminate the model service. The terminated model service will be removed"
+" from the list of model services."
+msgstr ""
+"모델 서비스는 주기적으로 스케줄러를 실행하여, 원하는 세션 수와 실제 대응하는 라우팅 수가 원하는 세션 수에 맞춰지도록 "
+"스케줄링합니다. 다만, 이 경우 Backend.AI 스케줄러에 부하가 가는 것은 불가피 합니다. 따라서 모델 서비스를 더 이상 "
+"사용하지 않는 경우라면, 모델 서비스를 종료하는 것이 좋습니다. 모델 서비스를 종료하려면, 제어 탭에서 휴지통아이콘을 클릭합니다. "
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:822]:
```suggestion
"사용하지 않는 경우라면, 모델 서비스를 종료하는 것이 좋습니다. 모델 서비스를 종료하려면, 제어 탭에서 휴지통 아이콘을 클릭합니다. "
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 7]
```diff
@@ -0,0 +1,961 @@
...
+msgid ""
+"The common reasons for creation failure and their solutions are as "
+"follows:"
+msgstr "생성이 안되는 이유 및 해결법은 대개 다음과 같습니다 :"
+
+#: ../../model_serving/model_serving.rst:334
+msgid ""
+"Insufficient allocated resources for the routing when creating the model "
+"service"
+msgstr "너무 적은 양의 자원을 모델 서비스 생성시 라우팅에 할당한 경우"
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:848]:
```suggestion
msgstr "모델 서비스 생성 시 너무 적은 양의 자원을 라우팅에 할당한 경우"
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 8]
```diff
@@ -0,0 +1,961 @@
...
+" name in the Model Service tab to view detailed information about the "
+"model service. From there, you can check the service endpoint in the "
+"routing information of the model service. If the “Open to Public” option "
+"is enabled when the service is created, the endpoint will be publicly "
+"accessible without any separate token, and end users can access it. "
+"However, if it is disabled, you can issue a token as described below to "
+"verify that the service is running properly."
+msgstr ""
+"모델 서비스를 성공적으로 실행한 경우, 상태는 ``HEALTHY`` 에 속하게 됩니다. 이 경우 모델 서비스 탭에서 해당하는 "
+"엔드포인트 명을 클릭해 모델 서비스의 상세 정보를 확인할 수 있습니다. 이후 모델 서비스의 라우팅 정보에서 서비스 엔트포인트를 "
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:887]:
```suggestion
"엔드포인트 명을 클릭해 모델 서비스의 상세 정보를 확인할 수 있습니다. 이후 모델 서비스의 라우팅 정보에서 서비스 엔드포인트를 "
```

File: docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po [hunk 9]
```diff
@@ -0,0 +1,961 @@
...
+"확인할 수 있는데, 이 엔드포인트는 서비스 생성시 외부 에 공개할 수 있는 \"Open to Public\" 값이 활성화 된 경우,"
```
Comment (agatha197) [inline docs/locale/ko/LC_MESSAGES/model_serving/model_serving.po:888]:
```suggestion
"확인할 수 있는데, 이 엔드포인트는 서비스 생성시 외부에 공개할 수 있는 \"Open to Public\" 값이 활성화 된 경우,"
```

**Review by agatha197 — CHANGES_REQUESTED** [review]

File: docs/model_serving/model_serving.rst [hunk 10]
```diff
@@ -0,0 +1,419 @@
...
+   :align: center
+   :alt: Routing page
+
+Click the token creation button located to the right of the generated
+token list in the routing information. In the modal that appears for
+token creation, enter the expiration date. The issued token will be
+added to the list of generated tokens. Click the copy icon in the token
+item to copy the token, and add it as the value of the following key.
+
+.. image:: edit-model-service-dialog.png
```
Comment (yomybaby) [inline docs/model_serving/model_serving.rst:377]: This image is not related to `Generating Token`.

**Review by yomybaby — CHANGES_REQUESTED** [review]

File: docs/model_serving/model_serving.rst (hunk 10 above)
Comment (yomybaby) [reply docs/model_serving/model_serving.rst:377]: image

**Comment by yomybaby** [comment]
@lizable 
On `Solution` parts, is it `Clear error and retry` not `Refresh`?
image

**Pull request merged**